src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitWorkerService.java

```diff
@@ -65,6 +66,28 @@ protected void saveResult(Network network, AbstractResultContext<ShortCircuitRun
                 ShortCircuitAnalysisStatus.COMPLETED.name());
     }
 
+    @Override
+    public void preRun(ShortCircuitRunContext resultContext) {
+        checkInconsistentVoltageLevels(resultContext);
+    }
+
+    private void checkInconsistentVoltageLevels(ShortCircuitRunContext resultContext) {
+        List<String> inconsistentVoltageLevels = new ArrayList<>();
+        Network network = resultContext.getNetwork();
+        if (network.getBusView() != null) {
+            network.getBusView().getBusStream().forEach(bus -> {
```

Comment:
If the voltage level has multiple buses, it will add multiple times the voltage level id to the list. Should we had it only once or should we add the bus is to the report ( ie : GUARBP6 - GUARBP6_11) ? 
To be validate with PO

Reply:
See with Pauline: we store only one report per voltage level even if we have multiple buses on one VL